app/src/types.ts

```diff
@@ -33,8 +32,16 @@ export interface PacketTableProps {
 
 export interface PacketsState {
     packets: Packet[]
-    error: SerializedError | null
-    packet: Packet
+    error:  null | Error
```

Comment:
So this `error` property is specifically for errors when fetching packets and `packetError` is specifically for errors fetching a single packet by id? Maybe they should be called `fetchPacketsError` and `fetchPacketError` or similar to make it clear?